frontend/console/playwright-report/index.html

```diff
@@ -65,4 +65,4 @@
qda+aYEEILp6EbOlkK5ufu0a638jUXh+T9+3no7F9k2T2vSnf10XayHZbVJGE94nAPh6jYx77S7BHCUqPxGykTImutH375aPJ9auzPCuuz+azu1mRbHQ8O/vt4QKdze5fozTP0s4+RwiKOOhFgnWdhNZHjtiLwPtHWNTWmmI/hE7XZkACijCGCTqXWmuCvbVG8bcmmL273ZrS6uWwKLA3KO4VhZC9NamOZHl4viNrPiSf9Z8ff+3hjwhyqhf3tnd0vQ8Q2dOI7ru75n9p3+pPWZUt8s7W6fJKkNfzcoSPXkXY53J8SvK6n0HQxPyWPtlb++JYA/I7gyGcRrwMUjzkDA4zCJ+Ofgb37jh5rP95GlSjGKR7BtkRgz+trC77Td/OmQJ3R+sQO2hcaVTBivF6KPd5oo7sbIxZekAV8FmXpSl3/1PZxNZVMxpVVQNwibVJut7ookG3S3eAuQ5iW9a6sXuSXxPOOGGEa75ISUK4ShUdwK+rrNQrczsKvlLGw/Hx9UG168IIIcbE10YxfHqpqZJ0BHxtpNWBdOvQPxRfnTZ8Koqw3fVgfG3EAe4J6YlHfW85hggDNekp5BuAr432AU9OR3uungD1QipOhBnIr00BAQvwSROpEMz3hKnWSGPxq6sJH22Eyc5Z93sCLBGICgw8dk6n/zd8FYjCAZj7jejPha8CMQKWGvMb8J4dX8XhmyYReUWZBl8F4hi+evPkuR+DXwXiFLQISvwCTwSwAvGDt9UvCmCdQdBgSCePfFeCFYgLeRTDXgDBCsThWy1Pg2MTrEA8BMNS67bFsIGlRfWFEKxzJiYn2P5Vvh/BpjQJ01REqcaYiDSSDIcDCPZGL64zOwrAKqzG59dG9PRVkf++KhwNX50iGMwj2oqYQ+m1v/JgeJWIhAzOAKdgbwi8OnEIe2xieJWIEVCTjrT32miDO5kyvzi9+pNEjMN55eTG+FB4dQUILDAljksk4PYaC8NJ2VUiATdpqN++5mToKpHCEK+p1/bwj8yuEkUUvtn3Gy2fi10ligRc3dIrwbOzq0QR/DmACL2iTMOuIcLwtRvz3Or7Mdg1RDiCr7c8XyZMxK4hIvCHB9TP4ETsGiJCAFz70f9U6OoMwnnpRaGrMwgHoJeBrs4Z2Dtq/d3jsHGlRfWFoGuns1GG6ZYqpHs6/CZ0vbz/ClBLAwQUAAAICADogiNZfe1Mz28DAACgFAAACwAAAHJlcG9ydC5qc29u3ZfJjts4EIZfxeBZbnATF7/BXHIKMIdBH4rFUqyxbBkS1UnQ8LsPaDtJe1ptRUZi9IxPpCTX8lX9JeqZbSlBhARs9cwA0wDNn223oa5nK3MoWJ+gSx/rLbGVsLJUXntnnHQFi0MHqW53bCW44v5Ba+e//QpW1Q31bPXX83H1R2QrpmUw1lKA0mjwPIYgHDs9+QGyA0ZPtEv9Q78nfEg9K1iiPp2s5NWbVpa+UlRFG4JVgBq1A8nz3+vUZLuRKhia1C9Su0hrWpz8LPbwiVjB9l37N2E6x4Drrt3Ww5YVrGnxnOEpi7EIm3pHbKUKhm0zbDOMw0s0muuyYLDbtel4JSfzWLAEn86rdkjYnpL/sidMFHNEkNbn2xu2St1ABeuozykcYUBKgOttDiU/dng8FFOEMJAXXAmo0HCKMgRX3kioqjuq2i+/BpAyTr0LQJG85misCFZFdMppEW4E9JnCpk6/ho8QpbkHn8ej1PP2maU2QXOM6bvJvBl2P7a8YFUDm6/HVb+p9/vz1W/+Dpn4d+lDWVpRBWUlcvQiGHLhUvr1ruqgT92Aaejo+ggYs7YUQMEqrkE7bkLUEo17Ub9+3X7uF5deZmr/zRCnWtyru7f4KCLrERQJAh2lDL6KJai5iK6K/1ZCwnvxLgjFiOAcONSeC6EdQaC5hK6p/1ZAprxLB/3mIQDEYxBeVzxC6QSWurocAts2Ds2U+EesLIM1iOAVBStMDDpIKV/V7Ym60C+qtltE2jft15z5zBHwKsCJull3n8IVk4DQVlYSCiVRKQ6afLgN0NUBMJeP8Nq+Cz6RvCXnhQgSPUofnRS38bkm/7l4nPgfvPtRcYzKk0BnK19VpjQwJvuJc/+YmSWRUVJYoYHnaoZSqtfzGjDVT7Q4e7lJ8XMOtF7Je7f0KBuFivtYklGOq8iVk1bPZfMTYp+BRnLP3wUaK6IjLagCzSNZI/zINJxAM63zGWSsvsv55/cKXZssP2G1i1Ry7SlKfSn0PEAnvu5HbCwl6KC9A+0DcR6jM4qPDuc8m7czBf6vmE6FynP3zQ/6ux9VR6GAVloooUkHFCC0Qzd+5BmHclXZM5lIdf+3+CgTlGARS4/EuSjRG8XtDCbXJD0TiePuvynoxwso2c9L/fzomjOrkQBE+TKCvJsXQsGo69rulOI/UEsBAj8DFAAACAgA6IIjWc7yDTLuAwAAmBwAABkAAAAAAAAAAAAAALSBAAAAADQyYjY3N2ViYTU2NGE5MGRiYjE4Lmpzb25QSwECPwMUAAAICADogiNZ42Ur7WQEAAA9JAAAGQAAAAAAAAAAAAAAtIElBAAAYTU1NzFmYjM3MmMwYzkxYjZlOGIuanNvblBLAQI/AxQAAAgIAOiCI1l2DDWwpgQAACYjAAAZAAAAAAAAAAAAAAC0gcAIAABhYWUwZGIxOTRmMGRhNTgxYzU0Zi5qc29uUEsBAj8DFAAACAgA6IIjWSD3w/lcBAAAhyAAABkAAAAAAAAAAAAAALSBnQ0AAGMzMGNkMzllMWM4N2Y5ZmY2NTZhLmpzb25QSwECPwMUAAAICADogiNZtmDNlkkFAACCLAAAGQAAAAAAAAAAAAAAtIEwEgAANDY1MjNlMTc0OGRlNTA0OWVkMjQuanNvblBLAQI/AxQAAAgIAOiCI1l97UzPbwMAAKAUAAALAAAAAAAAAAAAAAC0gbAXAAByZXBvcnQuanNvblBLBQYAAAAABgAGAJwBAABIGwAAAAA=";</script>
```

Comment:
I don't think this file should be in the repo at all 🤔

Reply:
I'll fix in a follow on PR

Reply:
Whoops thanks for catching that! I copied your gitignore change from adding the infra page.

Reply:
Whoops x2, I accidentally pushed a fix for this before seeing your second comment. I'll slack you